specify/specify7
# Fix preptype picklist not loading

Fixes #4941 



### Checklist

      and self-explanatory (or properly documented)

### Testing instructions




- Go to Data Entry
- Click on Collection Objects
- Expand the Preparations subview
- [ ] Verify there are no errors and the form loads as usual
- [ ] Additionally, make sure #4704 still works as expected
